src/App.js

```diff
@@ -17,13 +17,13 @@ class App extends React.Component {
           {/* <Header /> */}
           <Switch>
             <Route path="/" exact component={Warehouse} />
-            <Route path="/warehouse" exact component={Warehouse} />
+
+            <Route path="/warehouse" component={Warehouse} />
```

Comment:
Carried over "exact" attribute

```suggestion
            <Route path="/warehouse" exact component={Warehouse} />
```